docker/structs.go

```diff
@@ -0,0 +1,8 @@
+package docker
+
+import "github.com/docker/engine-api/types"
+
+// VolumeRequest is taken from
+// https://github.com/docker/engine-api/types/types.go
+type VolumeRequest types.VolumeCreateRequest
```

Comment:
If we can re-use all the structs here:

* if they cover all the API, no need for the shim
* if they only cover parts of the API, please insert shim.

You'll need to import that dependency; let me know when you're there, I need to show both you and @dseevr how to do this I'm guessing.